randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -1040,7 +1042,12 @@ Extra - room_id: [28]
   > Pickup (Energy Tank)
       All of the following:
           Speed Booster
-          Space Jump or Can Activate Pillar
+          Any of the following:
+              Space Jump or Can Activate Pillar
+              # Wall Jump and Shinespark straight up - https://youtu.be/Pvv2dYRZAj4
+              Wall Jump (Intermediate)
+              # Wall Jump and ignore the pillar - https://youtu.be/FJxCd30N2J4
+              Wall Jump (Advanced)
```

Comment:
I seem to recall that including tricks in this way will cause the randomizer to always pick one over the other. Feedback on how I could improve this is appreciated. If necessary we can remove one trick or the other.

Reply:
Yeah it'll always accept the lowest requirements. That being said I don't mind the "dual" documentation incase someone struggles maintaining the shinespark and just wants to beat their head against that other jump.

Reply:
Would we leave the logic in place for both or remove one of the trick requirements and just have 2 comments in that case?

Reply:
Just leave both requirements. I'd rather have more of the game documented than too little.